src/Client.cpp

```diff
@@ -1669,6 +1702,14 @@ bool CClient::OnTagMessage(CTargetMessage& Message) {
 bool CClient::OnTextMessage(CTextMessage& Message) {
     CString sTargets = Message.GetTarget();
 
+    // Check for empty PRIVMSG
```

Comment:
This belongs to a separate commit, this is not related to away-notify nor chghost